lib/scrabble.rb

```diff
@@ -1,18 +1,29 @@
 class Scrabble
-
-  def score(word)
-    1
+  attr_reader :point_values
+  def initialize
+    @point_values = {
+     "A"=>1, "B"=>3, "C"=>3, "D"=>2,
+     "E"=>1, "F"=>4, "G"=>2, "H"=>4,
+     "I"=>1, "J"=>8, "K"=>5, "L"=>1,
+     "M"=>3, "N"=>1, "O"=>1, "P"=>3,
+     "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
+     "U"=>1, "V"=>4, "W"=>4, "X"=>8,
+     "Y"=>4, "Z"=>10
+   }
   end
 
-  def point_values
-    {
-      "A"=>1, "B"=>3, "C"=>3, "D"=>2,
-      "E"=>1, "F"=>4, "G"=>2, "H"=>4,
-      "I"=>1, "J"=>8, "K"=>5, "L"=>1,
-      "M"=>3, "N"=>1, "O"=>1, "P"=>3,
-      "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
-      "U"=>1, "V"=>4, "W"=>4, "X"=>8,
-      "Y"=>4, "Z"=>10
-    }
-  end
-end
+   def score(word)
+     point_values[word.upcase]
```

Comment:
This line appears at the top of both methods.  It would take the whole word, such as "apple", and try to get the key of "APPLE" from the point_values hash. The result of this would be nil since there is no "APPLE" in the hash. Additionally, you don't save the return of this action, so you can't do anything with it later on.